datajoint_utilities/dj_worker/worker.py

```diff
@@ -130,18 +130,20 @@ def __init__(
         self._sleep_duration = sleep_duration
         self._max_idled_cycle = max_idled_cycle if RETURN_SUCCESS_COUNT else -1
         self._db_prefix = [db_prefix] if isinstance(db_prefix, str) else db_prefix
-        
+
         # Handle backward compatibility
         if remove_stale_reserved_jobs is not None:
             warnings.warn(
                 "The 'remove_stale_reserved_jobs' parameter is deprecated and will be removed in a future version. "
                 "Use 'stale_timeout_hours' instead.",
                 DeprecationWarning,
-                stacklevel=2
+                stacklevel=2,
             )
             # If remove_stale_reserved_jobs is False, set time limit to 0 (disabled)
             # If True, keep the default 24 hours
-            self._stale_timeout_hours = 0 if not remove_stale_reserved_jobs else stale_timeout_hours
+            self._stale_timeout_hours = (
+                0 if not remove_stale_reserved_jobs else stale_timeout_hours
```

Comment:
```suggestion
                stale_timeout_hours if remove_stale_reserved_jobs else 0
```

Reply:
Consider replacing `remove_stale_reserved_jobs` with `remove_stale_jobs`

Reply:
Keeping `remove_stale_reserved_jobs` argument name to maintain backward compatibility - this argument is deprecated